### PR TITLE
Padding transformer

### PR DESCRIPTION
**Pull request by ABostrom**
Reference Issues/PRs

#6
What does this implement/fix? Explain your changes.

Implements the Padder Transformer
Does your contribution introduce a new dependency? If yes, which one?

no
Any other comments?

This is in response to the list on PR's for beginners,

Thanks for contributing!
-->

### CODE REVIEW

**Comment by ABostrom** [comment]
Started the PR @mloning just so that you can see I've begun work on this. Don't think it'll take me very long as it's quite similar to the truncator.

**Comment by ABostrom** [comment]
@mloning okay so that was super easy...

Hopefully tests should pass, and with any of your suggestions should be good to go.

**Comment by ABostrom** [comment]
@mloning Implemented all your suggestions. Np.full seems to work fine. I set the dtype to np.float, because i can't see a use case where we don't have that as our type. Was getting weird results if i input an int as the fill type without it.

File: sktime/transformers/series_as_features/tests/test_Padder.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+from sktime.datasets.base import _load_dataset
+from sktime.transformers.series_as_features.padder import \
+    PaddingTransformer
+# from sklearn.ensemble import RandomForestClassifier
+from sktime.utils.data_container import tabularize
+
+# import pandas as pd
+
+
+def test_truncation_transformer():
```
[the remark this answers is not in the document]
Comment (ABostrom) [reply sktime/transformers/series_as_features/tests/test_Padder.py]: To respond to your second point. I agree. I'm currently writing a transformer for TSML package, which will behave in a similar manner to these other transformers. So i think a sub foler / module is probably a good idea. 

Might leave that refactor to you though :)

**Comment by mloning** [comment]
@ABostrom looks great, thanks!

Had to update with newest changes on dev, will merge once all tests pass again!
